<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QGroupBox" name="scrcpyCheck">
         <property name="title">
          <string>Custom scrcpy path</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="scrcpyEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="adbCheck">
         <property name="title">
          <string>Custom adb path</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="adbEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="scrcpyServerCheck">
         <property name="title">
          <string>Custom scrcpy-server path</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="scrcpyServerEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Devices</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="coherenceCheck">
         <property name="text">
          <string>Coherence mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="customScreenSizeCheck">
         <property name="text">
          <string>Custom default window size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item row="1" column="2">
           <widget class="QSpinBox" name="heightEdit">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>2400</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="widthEdit">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>1080</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>adbCheck</tabstop>
  <tabstop>adbEdit</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>scrcpyCheck</tabstop>
  <tabstop>scrcpyEdit</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>scrcpyServerCheck</tabstop>
  <tabstop>scrcpyServerEdit</tabstop>
  <tabstop>toolButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>editAdbPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>editScrcpyPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>editScrcpyServerPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adbCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>toggleCustomAdbPath(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrcpyCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>toggleCustomScrcpyPath(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrcpyServerCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>toggleCustomScrcpyServerPath(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adbEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setAdbPath(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrcpyEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setScrcpyPath(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrcpyServerEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setScrcpyServerPath(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customScreenSizeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>coherenceCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>setCoherenceMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editAdbPath()</slot>
  <slot>editScrcpyPath()</slot>
  <slot>editScrcpyServerPath()</slot>
  <slot>toggleCustomAdbPath(bool)</slot>
  <slot>toggleCustomScrcpyPath(bool)</slot>
  <slot>toggleCustomScrcpyServerPath(bool)</slot>
  <slot>setAdbPath(QString)</slot>
  <slot>setScrcpyPath(QString)</slot>
  <slot>setScrcpyServerPath(QString)</slot>
  <slot>setCoherenceMode(bool)</slot>
 </slots>
</ui>
